src/desktop/apps/artist/client/views/header.coffee

```diff
@@ -13,12 +13,18 @@ module.exports = class ArtistHeaderView extends Backbone.View
     @setupShareButtons()
     @setupFollowButton()
     @updateCurrentItem()
+    # TODO: ARTIST_MARKET_DATA_TEST remove after test closes
+    @setupMarketDataSummary()
+
     @$window = $ window
     @$('a').click @navClick
 
   setupShareButtons: ->
     new ShareView el: @$('.artist-share')
 
+  setupMarketDataSummary: ->
+    require('./market_data_summary.js').default.renderArtistMarketDataSummary(@model.id)
```

Comment:
Not sure that was the best way to embed stuff from Reaction here. But that works.

Reply:
Should this run for users who don't have the lab feature (or are in the control group)? I'm wondering if there should be an early-return or guard, or if this silently doesn't do anything if the container element doesn't exist on the page, for instance.